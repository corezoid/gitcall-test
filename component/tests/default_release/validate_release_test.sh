#!/usr/bin/env bash
$CZ valid
