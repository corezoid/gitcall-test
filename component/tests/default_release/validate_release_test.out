level=info msg="Loading release from current dir"
level=info msg="Validating release schema"
level=info msg="Validating gitcall: main.json schema"
level=info msg="Release valid"
